apps/api/src/allowed-activity/entity/allowed-activity.entity.ts

```diff
@@ -13,6 +13,9 @@ export class AllowedActivity extends CustomBaseEntity {
   @ManyToOne(() => Occupation, occupation => occupation.allowedActivities, { nullable: false })
   occupation: Occupation;
 
-  @ManyToOne(() => CareActivity, careActivity => careActivity.occupations, { nullable: false })
+  @ManyToOne(() => CareActivity, careActivity => careActivity.occupations, {
+    nullable: false,
+    cascade: true,
```

Comment:
Just given the branch name, I am wondering if this is intended to be used with the delete operation?

Reply:
Good one! Fixed 👍 Added onDelete and onUpdate flag